tools/lib/helpers.py

```diff
@@ -5,12 +5,16 @@
 
 # regex patterns
 class RE:
-  DONGLE_ID =  r'(?P<dongle_id>[a-z0-9]{16})'
```

Comment:
this block is pretty dense now, and it's not obvious from glancing at it that there are bases and groups. I suggest putting all the bases at the top and separating out the grouped bases with a blank line